antora/docs/modules/ROOT/pages/trusted_tasks.adoc

```diff
@@ -36,7 +42,7 @@ would result in such a warning.
 The process of adding Tasks to the list of trusted Tasks is described
 xref:trusting_tasks.adoc[here].
 
-== Trusted Task list
+=== Example Structure
```

Comment:
FYI, as mentioned on line 61, the `task-bundles` key and its contents is deprecated. It says we're removing it. This might be a good time. Wdyt @lcarva ?

Not a blocker for merging, but you could consider doing it while you're looking at this, maybe in a commit in this PR, or in another PR.

Reply:
No concerns. Nothing should be using it anymore. I do see [some reference](https://github.com/search?q=repo%3Aenterprise-contract%2Fec-policies%20task-bundles&type=code) but they seem to be leftover tests. We should probably remove those too at some point.

Reply:
Thank you for confirming!
I tried deleting the leftover references in [this commit](https://github.com/enterprise-contract/ec-policies/pull/1364/commits/0a2530394e545b7b9ef9579c7dd674b1ba277be0).

Reply:
The CI check is complaining about the reduced test coverage ([here](https://github.com/enterprise-contract/ec-policies/actions/runs/14739175539/job/41375202403?pr=1364#step:6:9)), which is the result of the removal of the test-bundle-related tests. Should we ignore this CI check failure?

Reply:
I agree with you, let's wind this PR back to a docs change only. Thanks for the suggestions!!